plugins/inputs/nightscout/README.md

```diff
@@ -0,0 +1,1141 @@
+# Nightscout Input Plugin
+
+The nightscout plugin collects blood glucose related telemtry from a nightscout site. Not all available data is collected at this time, simply the basics needed for active monitoring.
+
+### Configuration:
+
+```toml
+  ## This is a list of nightscout sites to pull data from.
+  ##  example: servers = ["https://my.nightscout.site/api/v1/entries"]
+  servers = []
+  ## The person is the owner of the nightscout site.
+  # owner = ""
+
+  ## The secret is the SHA-1 hash of the api secret used to access the site.
+  # secret = ""
+
+  ## The count is the number of results to pull at a time. Unless you have a specific need, this should remain as 1.
+  # count = "1"
+
+  ## The token is the unique token that matches a role within your nightscout site(s).
+  token = ""
+```
+
+### Collected Measurements:
+
+- Type
+- DateString
+- Date
+- SGV
+- Direction
+- Noise
+- Filtered
+- Unfiltered
+- RSSI
+
+### Tags:
+
+- All measurements have the following tags:
+    - owner
+
+### Example Output:
+
+```
+nightscout,host=localhost,owner=bob noise=0i,filtered=0i,rssi=0i,direction_num="-1",dateString="2020-04-04T14:12:51.000Z",date=1586009571000i,direction="FortyFiveDown",unfiltered=0i,type="sgv",sgv=277i 1586009730000000000
+```
+
+### Example Grafana Dashboard
```

Comment:
I wonder if it would make more sense to put the dashboard in a separate file instead of inlining in the readme?